[DOCS] Doc privs needed to import rules with actions (#1762)
elastic/security-docs: docs/getting-started/detections-req.asciidoc

^1^ *NOTE*: If you're upgrading to {stack} 8.0.0 or later, users should have privileges for the `.alerts-security.alerts-<space-id>` AND `.siem-signals-<space-id>` indices. If you're newly installing the {stack}, then users do not need privileges for the `.siem-signals-<space-id>` index.

a| {kib} space `All` privileges for the `Security` feature (refer to
{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])

> *NOTE:* Your role will need additional `Action and Connectors` feature privileges (**Management → Action and Connectors**) to manage rules with actions and connectors:

Reviewer: ```suggestion
*NOTE:* You need additional `Action and Connectors` feature privileges (**Management → Action and Connectors**) to manage rules with actions and connectors:
```

> * To provide full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, you can edit rule actions but will have limited connector management capabilities. For example, `Read` privileges allow you to add or remove an existing connector from a rule, but not to create a new one.

Author: Thanks for the edits, @jmikell821 ! In the last sentence, I'm trying to convey that if a rule has an action and connector defined, a user with `Read` privs can choose a different action and connector, or remove both. What they can't do is create a new connector or edit the settings of an existing connector.

> * To provide full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, you can edit rule actions but will have limited connector management capabilities. For example, `Read` privileges allow you to add or remove an existing connector from a rule, but not to create a new one.

Author: In the last sentence, I'll change "one" to "connector" so it's less ambiguous.

> * To provide full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, your role can edit rule actions but has limited connector management capabilities. For example, `Read` privileges would allow you to add or remove an existing connector from the rule, but not to create a new one.

Reviewer 2: ```suggestion
* To provide full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, your role can edit rule actions but has limited connector management capabilities. For example, `Read` privileges allow you to add or remove an existing connector from the rule, but not to create a new one.
```

> * To provide full access to editing rule actions and managing connectors, give your role `All` privileges. With `Read` privileges, your role can still edit rule actions but will have limited connector management. For example, your role can add or remove an existing connector from the rule, but it cannot create a new one.

Reviewer: ```suggestion
* To enable full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, your role can edit rule actions but has limited connector management capabilities. For example, `Read` privileges would allow you to add or remove an existing connector from the rule, but not to create a new one.
```

Author: Good with all of these, but am leaning towards keeping "provide" over "enable" since the latter makes me think of a toggle or a binary setting.

> * To import rules with actions and connectors, your role needs at least `Read` privileges. If you are importing rules without actions or connectors, your role does not require `Action and Connectors` feature privileges.

Reviewer: ```suggestion
* To import rules with actions and connectors, your role needs at least `Read` privileges. To import rules without actions or connectors, your role does not require `Action and Connectors` feature privileges.
```
